tests/unittests/tests/01-run.py

```diff
@@ -7,13 +7,17 @@
 # directory for more details.
 
 import os, signal, sys, subprocess
-from pexpect import spawn, TIMEOUT, EOF
+import pexpect
 
 DEFAULT_TIMEOUT = 5
 
 def main():
     env = os.environ.copy()
-    child = spawn("make term", env=env, timeout=DEFAULT_TIMEOUT)
+    if sys.version_info < (3,):
```

Comment:
I think this will yield other problems... This is taken as is from the [documentation](https://pexpect.readthedocs.org/en/latest/api/pexpect.html#spawn-class) (with the added version check):

> Example log to stdout:
>
> ```py
> # In Python 2:
> child = pexpect.spawn('some_command')
> child.logfile = sys.stdout
> 
> # In Python 3, spawnu should be used to give str to stdout:
> child = pexpect.spawnu('some_command')
> child.logfile = sys.stdout

Also the documentation doesn't really state what to set this parameter.